src/functionalTest/resources/scenarios/RIA-5912-submit-application-upload-bail-evidence.json

```diff
@@ -0,0 +1,61 @@
+{
+  "description": "Submit the application with bail evidence attached (pre-submit REST call)",
+  "request": {
+    "uri": "/bail/ccdAboutToSubmit",
+    "credentials": "AdminOfficer",
+    "input": {
+      "id": 1111,
+      "eventId": "submitApplication",
+      "state": "applicationStarted",
+      "caseData": {
+        "template": "minimal-application-started.json",
+        "replacements": {
+          "uploadTheBailEvidenceDocs":
+          [
+            {
+              "id": "3",
+              "value": {
+                "document": {
+                  "document_url": "http://dm-store:8080/documents/1488f4de-c9fe-4f3e-a6b6-ee032ca759c0",
+                  "document_binary_url": "http://dm-store:8080/documents/1488f4de-c9fe-4f3e-a6b6-ee032ca759c0/binary",
+                  "document_filename": "evidence.PDF"
+                },
+                "description":"test evidence description"
+              }
+            }
+          ]
+        }
+      }
+    }
+  },
+  "expectation": {
+    "status": 200,
+    "errors": [],
+    "caseData": {
+      "template": "minimal-application-started.json",
+      "replacements": {
+        "uploadTheBailEvidenceDocs":
```

Comment:
On the expectation, this should be applicantDocumentsWithMetadata instead of uploadTheBailEvidenceDocs. 

REASON: By the end of the UploadBailEvidenceDocumentHandler class, the document you uploaded into uploadTheBailEvidenceDocs will be added to the applicantDocumentsWithMetadata collection.